distros/spring/src/main/resources/application-dev.yml

```diff
@@ -1,12 +1,6 @@
-spring:
-  rabbitmq:
-    user: ion
-    password: ion
-    host: localhost
-    port: 5672
-    virtualHost: /
-    exchange: transformation
-    routingKey: request
+endpoints:

```

Comment:
@figliold - ❗Can you add the copyright header to this file, please?

Reply:
added